projects/packages/identity-crisis/src/class-rest-endpoints.php

```diff
@@ -62,6 +63,17 @@ public static function initialize_rest_api() {
 				),
 			)
 		);
+
+		// Fetch URL verification secret.
+		register_rest_route(
+			'jetpack/v4',
+			'/identity-crisis/fetch-url-secret',
```

Comment:
Not a blocker: Do you think we could make the route more RESTfull by avoiding the verb (`fetch`) in its name? aka `GET /jetpack/v4/identity-crisis/url-secret`

Reply:
Good point, updated: https://github.com/Automattic/jetpack/pull/32005/commits/bb6c956dd49a6cff5bd8005c14e2474404fa25bb
@bindlegirl please note that the fetching endpoint URL has changed to `/jetpack/v4/identity-crisis/url-secret`